src/inmanta/data/model.py

```diff
@@ -661,3 +661,12 @@ class Notification(BaseModel):
     uri: str
     read: bool
     cleared: bool
+
+
+class Source(BaseModel):
+    """Model for source code"""
+
+    hash: str
+    file_name: str
```

Comment:
@bartv I wonder if it would not make more sense to not serve the file_name at all and instead have boolean field 'is_compiled' 

We don't use the file_name anyways.

Reply:
fine for me

Reply:
done